junebug/logging_service.py

```diff
@@ -72,13 +70,13 @@ def _log_to_file(self, event):
             data['instance'] = repr(failure.value)
             data['stack'] = failure.stack
 
-        data['message'] = log.textFromEventDict(event)
-
         self.logfile.write(json.dumps(data))
 
     def __call__(self, event):
         if self.log_context_sentinel in event:
             return
+        if self.worker_id not in event.get('system', ''):
+            return
```

Comment:
The situation I'm worried about is if `worker_id` is `foo` and `system` is `foobar`. Then `worker_id` is in `system`, but not the way we want. It's very much a corner case. Sorry, should have made that clearer earlier.